src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../types';
+import { findPersonByName } from '../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+type ParentLinkProps = {
+  parentName: string | null;
+  parent: { slug: string } | null;
+  className?: string;
+};
+
+const ParentLink: React.FC<ParentLinkProps> = ({
+  parentName,
+  parent,
+  className,
+}) =>
+  parent ? (
+    <Link to={`/people/${parent.slug}`} className={className || ''}>
+      {parentName}
+    </Link>
+  ) : (
+    <>{parentName || '-'}</>
+  );
```

Comment:
The ParentLink component is defined as a functional component using React.FC. This is correct, but note that using React.FC is optional and sometimes discouraged due to implicit children prop and other reasons. However, if it aligns with your project's conventions, it's fine to use.